vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -694,4 +694,13 @@
     <Error Condition="'$(Source)'==''" Text="To generate a pre-processed file, please specify a source file with msbuild /p:Source=nameOfTheSource.cpp" />
     <CL PreprocessToFile="true" Sources="$(Source)" AdditionalIncludeDirectories="%(ClCompile.AdditionalIncludeDirectories)" PreprocessorDefinitions="%(ClCompile.PreprocessorDefinitions)" AdditionalOptions="%(ClCompile.AdditionalOptions) /d1PP" PreprocessKeepComments="true" PreprocessOutputPath="$(Source.Replace('.cpp', '.pp'))" />
   </Target>
+
+  <!-- https://github.com/microsoft/microsoft-ui-xaml/issues/3133 -->
+  <Target Name="InjectForcedXamlIncludes" BeforeTargets="CompileXamlGeneratedFiles" >
```

Comment:
Typo:
```suggestion
  <Target Name="InjecttForcedXamlIncludes" BeforeTargets="CompileXamlGeneratedFiles">
```

Reply:
Sorry, please ignore this one, I remain baffled by GH not updating the code in comments on the discussion tab like Azure DevOps does....